tests/unit/database/test_topic.py

```diff
@@ -37,7 +37,8 @@ def test_normal_behaviour_works(self, test_data_path):
             responses.GET,
             (
                 "https://www.ncbi.nlm.nih.gov/nlmcatalog?"
-                "term=Trauma Surg Acute Care Open[ta]&report=xml&format=text"
```

Comment:
Is passing from `[ta]` to `[Journal]` the only change needed to use full titles?

Reply:
Yes